js/aside-script.js

```diff
@@ -211,7 +220,18 @@ function UpdateLocale()
 	);
 }
 
-function AddCollection(collection)
+
+function ReloadCollections(collections,thumbnails){
+				document.querySelector(".tabsAside section h2").removeAttribute("hidden");
+				document.querySelectorAll(".tabsAside section > div").forEach(i => i.remove());
+
+				if (document.querySelector(".tabsAside.pane section > div") == null)
```

Comment:
We don't, nice catch ! Since we delete all the sets right above that, this if case should always happen.

Probably an unchecked copy-paste from the old "loadData" mechanism, that did not delete previous collections, my bad.
https://github.com/XFox111/TabsAsideExtension/blob/4559b25739091949b7b127d82b207cdeceb7ec0f/js/aside-script.js#L193-L198